src/components/common/GreenhouseEffectSettings.js

```diff
@@ -47,12 +52,37 @@ const GreenhouseEffectSettings = () => {
     dispatch(setAlbedoValues({ [key]: value }));
   };
   const totalEffectValue =
-    computeGreenhouseEffect({ methane, carbonDioxide }) * 100;
+    computeGreenhouseEffect({ methane, carbonDioxide, simulationMode }) * 100;
 
-  const albedo = computeAlbedo(albedoValues).albedo * 100;
+  const albedo =
+    computeAlbedo({ ...albedoValues, simulationMode }).albedo * 100;
+
+  const isMarsOrVenus =
+    simulationMode === SIMULATION_MODES.MARS.name ||
+    simulationMode === SIMULATION_MODES.VENUS.name;
 
   // greenhouse effect settings should only affect the simulation on flux mode
-  const disabled = !isPaused || radiationMode !== RADIATION_MODES.FLUXES;
+  const disabled =
+    !isPaused || radiationMode !== RADIATION_MODES.FLUXES || isMarsOrVenus;
+
+  // adjust CO2 and H2O concentration sliders max/min points if planet other than earth is selected
+  const CARBON_DIOXIDE_CONCENTRATION_MAX_VALUE = isMarsOrVenus
+    ? CARBON_DIOXIDE_CONCENTRATION_MAX_VALUE_ON_MARS_OR_VENUS
+    : CARBON_DIOXIDE_CONCENTRATION_MAX_VALUE_DEFAULT;
+
+  const WATER_CONCENTRATION_MIN_VALUE = isMarsOrVenus
+    ? WATER_CONCENTRATION_MIN_VALUE_ON_MARS_OR_VENUS
+    : WATER_CONCENTRATION_MIN_VALUE_DEFAULT;
+
+  // used to transofrm the label on the CO2 slider when either Mars or Venus are selected
+  // by default '965000' would be shown; this adds a comma separator so that it's 965,000
+  const formatCarbonDioxideLabel = (num) => {
+    const numString = num.toString();
+    if (isMarsOrVenus) {
+      return `${numString.slice(0, 3)},${numString.slice(3)}`;
+    }
+    return num;
+  };
```

Comment:
Perhaps you could use a library. I've found https://www.npmjs.com/package/numeral but it's quite old (but still used).

Reply:
you're right, this is not the best. but since it's just one number we're formatting, maybe not worth adding a library.